charts/kube-prometheus-stack/hack/requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyYAML==6.0.1
 requests==2.31.0
-jsonnet
+# jsonnet # installed via source
```

Comment:
As I know, pip also support git repos as source and allow them to store inside requirements.txt

ref: https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source

If possible, I would prefer this, including a fixed sha sum, e.g.: `jsonnet @ git+https://github.com/owner/repo@41b95ec`

The benefit is that if it works, it's more standardized and other integration like renovate can pickup this dependency and bump, if necessary.

Reply:
I just saw that, didn't know this is possible.
I added the change in my latest commit.

Reply:
This worked, according to the pipeline.